sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/v2Commands.scala

```diff
@@ -1142,3 +1142,42 @@ case class AlterTableRenameColumn(
   override protected def withNewChildInternal(newChild: LogicalPlan): LogicalPlan =
     copy(table = newChild)
 }
+
+/**
+ * The logical plan of the ALTER TABLE ... CHANGE COLUMN command.
```

Comment:
nit: let's put the SQL standard syntax in the doc `ALTER TABLE ... ALTER COLUMN`

Reply:
Updated